docker/local/README.md

```diff
@@ -0,0 +1,39 @@
+## Turbinia local stack using Docker
+Turbinia can be run locally without any Cloud components using Docker. It will use Redis, Celery and local disk to store data and perform message broker functionality.
+
+### Caveats
+rawdisk: As Turbinia uses the loop device to mount different types of evidence (eg raw disks) the host operating system should support the loop device. Linux is currently the only OS that supports the processing of raw disks. It is possible to process different evidence types, eg compresseddirectory, using the local stack on OSX or Windows.
+
+googleclouddisk: Turbinia as local stack can currently not process Google Cloud disks.
+
+### Running
+
+#### Step 1
+Checkout the Turbinia source code. If you intend to start developing please fork the repository on github first and check out your own forked instance.
+```
+$ git clone https://github.com/google/turbinia.git
+$ cd turbinia
+```
+#### Step 2
+Copy the configuration template for the local stack to the ```./conf`` folder (create the folder if it does not exist).
```

Comment:
I wonder if we should use /etc/turbinia on the host as well?

Reply:
so this is a relative path to a folder that contains the turbinia configuration. The docker-compose file maps this into the container in the right location. So instead of ./conf you would prefer ./etc, correct?

Reply:
Yeah, I know it's set up here as a relative path, but I was wondering if we should keep these in a more permanent location on the host instead of wherever the code is checked out to.  I was proposing an absolute path of /etc/turbinia, but I'm not sure if that's a great idea or not.  I'm just trying to figure out what it might look like in a more productionized setup.

Reply:
ah, got it. Yeah, in a production environment it would definitely point to an absolute path. I don't think I would take the host /etc/ as that's really for configuration files of that host. Probably I would store it on a NFS mount (something like a criticalvolume...).